tensorflow/lite/kernels/internal/reference/div.h

```diff
@@ -172,11 +212,92 @@ inline void BroadcastDivSlow(const ArithmeticParams& params,
                              const int8_t* input2_data,
                              const RuntimeShape& unextended_output_shape,
                              int8_t* output_data) {
-  BroadcastDivSlowQuantized<int8_t, N>(
-      params, unextended_input1_shape, input1_data, unextended_input2_shape,
-      input2_data, unextended_output_shape, output_data);
+  if (!params.is_quantized) {
```

Comment:
Same, why no use the existing templated template <typename T, int N = 5> void BroadcastDivSlow function for the unquantized path?